src/components/CharacterList.js

```diff
@@ -1,16 +1,130 @@
-import React, { useEffect, useState } from "react";
+import React, {
+  useState,
+  useEffect
+} from "react";
+import axios from "axios";
+import CharacterCard from "./CharacterCard";
+import styled from "styled-components";
+
+const StarDiv = styled.div `
+display: flex;
+/* border: 3px solid red; */
+width: 80%;
+height: 60vh;
+margin: 0 auto;
+flex-wrap:wrap;
+justify-content: space-evenly;
+
+`;
 
 export default function CharacterList() {
-  // TODO: Add useState to track data from useEffect
+  const [items, setStar] = useState([]);
```

Comment:
best practice is to name your setter functions after whatever your calling your state for example
``` const [items, setItems] = useState([]); ```